administrator/components/com_banners/helpers/html/banner.php

```diff
@@ -111,6 +111,6 @@ public static function pinned($value, $i, $enabled = true, $checkbox = 'cb')
 			),
 		);
 
-		return JHtml::_('jgrid.state', $states, $value, $i, 'banners.', $enabled, true, $checkbox);
+		return JHtml::_('jGrid.state', $states, $value, $i, 'banners.', $enabled, true, $checkbox);
```

Comment:
I fixed it on all occurrences, we have the same problem for contentLanguages sometime with and sometimes without uppercase "L". So we might need to make a general decision what we will do with class names. If we are going to use the composer autoloader it is needed that the names match exactly.